tests/test_nuget.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2022 Google LLC
+#
```

Comment:
Adding the missing copyright from upstream https://github.com/NuGet/NuGet.Client/blob/eea4636696cc7b6a3e682797c3dd717ad25fd7c9/test/NuGet.Core.Tests/NuGet.Versioning.Test/VersionComparerTests.cs
```suggestion
# Copyright (c) .NET Foundation. All rights reserved.
```